build.gradle

```diff
@@ -29,7 +29,7 @@ task clean(type: Delete) {
 
 ext {
     // sdk and tools
-    minSdkVersion = 16
+    minSdkVersion = 21
```

Comment:
Why do you always make unnecessary changes?

Reply:
that was required for some  xml attributes